packages/i18n/src/lang/fr.json

```diff
@@ -1535,6 +1535,10 @@
     "defaultMessage": "L'une ou les deux étapes données ne peuvent être interchangées.",
     "description": "Error message that one of the assessment steps cannot have its sort order changed."
   },
+  "l0aGCz": {
+    "defaultMessage": "Notification dans l’application",
```

Comment:
I don't think _Notification dans l’application_ is an equivalent to **In-app** in English since the English string doesn't include anything about notification.

@gobyrne what about _Intégré à l'application_?

- REF 1: https://www.btb.termiumplus.gc.ca/tpv2alpha/alpha-eng.html?lang=eng&i=1&srchtxt=IN-APP+PURCHASE&codom2nd_wet=1#resultrecs
- REF 2: https://fr.bab.la/dictionnaire/anglais-francais/in-app

Reply:
Whatever the choice on French copy, the titles should probably be in columns to account for differences in text length between languages.

## Screenshot
<img width="873" alt="Screen Shot 2024-04-29 at 11 01 35" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/83374a35-9ce4-4abf-b04b-48c72603fa05">